doc/source/cluster/quickstart.rst

```diff
@@ -0,0 +1,240 @@
+.. _ref-cluster-quick-start:
+
+Quick Start Cluster Autoscaling Demo
+====================================
+
+This quick start demonstrates the capabilities of Distributed Ray. Using Distributed Ray, we'll take a sample application designed to run on a laptop and scale it up in the cloud. Ray will launch clusters and scale Python with just a few commands.
+
+About the demo
+--------------
+
+This demo will walk through an end-to-end flow:
+
+1. Create a (basic) Python application.
+2. Launch a cluster on a cloud provider.
+3. Run the application in the cloud.
+
+Requirements
+~~~~~~~~~~~~
+
+To run this demo, you will need:
+
+* Python installed on your development machine (typically your laptop), and
+* an account at your preferred cloud provider (AWS, Azure or GCP).
```

Comment:
I'm worried that this gives off an impression that Ray doesn't consider kubernetes as first class. Can you please add a ton of references to the Kubernetes page in the quick start/intro?

Reply:
Sure, @DmitriGekhtman will do that in a followup PR.